### PR TITLE
Fix view errors

### LINKED ISSUE #57: Fix Me View
There are a view things to fix in files and especially the Me view :

- There are 2 ts errors (l.233 and l.271) in Me view
- The tournament and the rule href are currently not provided (l.231, l.239, l.269, l.277)
- Unpaid tournament are not flagged as unpaid. This is because of a divergence between front and back. PaymentStatus.NOT_PAID Should be equal to 'NOTPAID'. Verifications will be required on the other values.
- Team detail currently isn't a view. So we should redirect to the teams recap instead for now (l.222, l.262)

### PR DESCRIPTION
**Pull request by KwikKill**
Fix #57
ESlint error is deferred to the parent tournament branch.

### CODE REVIEW

**Review by istar410 — APPROVED** [review]
LGTM

File: src/views/Me.vue [hunk 1]
```diff
@@ -228,15 +228,18 @@ const editField = (field: string) => {
                   <a
                     :class="{ [`bg-red-600`]: inscriptions.unpaid[inscription[1].team.id], [`bg-green-600`]: !inscriptions.unpaid[inscription[1].team.id] }"
                     class="center rounded p-2 font-bold text-white transition duration-150 ease-in-out hover:ring hover:ring-pink-500"
-                    href=""
+                    :href="`/tournament/${inscription[1].team.tournament.id }?s=teams`"
                   >
                     {{ inscriptions.unpaid[inscription[1].team.id] ? 'Terminer l\'inscription' : (parseInt(inscription[1].team.players[0]) === user.id || inscription[0] === "manager") ? 'Gérer l\'équipe' : 'Voir l\'équipe' }}
```
Comment (Dyskal) [inline src/views/Me.vue:233]: There still are Typescript errors here

File: src/views/Me.vue [hunk 2]
```diff
@@ -258,23 +261,28 @@ const editField = (field: string) => {
...
           >
             <div class="my-1 hidden md:block">
               <div class="m-1 flex h-8 flex-1 flex-col justify-center">
                 <div>
                   <a
                     class="center rounded bg-green-600 p-2 font-bold text-white transition duration-150 ease-in-out hover:ring hover:ring-pink-500"
-                    href=""
+                    :href="`/tournament/${inscription[1].team.tournament.id }?s=teams`"
                   >
                     {{ (parseInt(inscription[1].team.players[0]) === user.id || inscription[0] === 'manager') ? 'Gérer l\'équipe' : 'Voir l\'équipe' }}
```
Comment (Dyskal) [inline src/views/Me.vue:274]: Here too

File: src/views/Me.vue (hunk 1 above)
Comment (KwikKill) [reply src/views/Me.vue:233]: There are still 2 Typescript errors due to wrong model changes in the tournament branch. 
This was introduced and reported in the https://github.com/InsaLan/frontend-insalan.fr/pull/54#discussion_r1401652889 PR and will be fixed there.

File: src/views/Me.vue (hunk 2 above)
Comment (KwikKill) [reply src/views/Me.vue:274]: Same as previous reply

**Review by Dyskal — COMMENTED** [review]
Shouldn't `règlement du tournois` be `Règlement du tournoi`?

**Comment by KwikKill** [comment]
> Shouldn't `règlement du tournois` be `Règlement du tournoi`?

Indeed, it's better like this. Fixed now :+1: !

**Review by Dyskal — APPROVED** [review]

**Pull request merged**